greenMap: '-1.440000, 1.440000, 0; 
-1.120000, 1.440000, 0; 
-0.800000, 1.440000, 0; 
-0.480000, 1.440000, 0; 
-0.160000, 1.440000, 0; 
0.160000, 1.440000, 0; 
0.480000, 1.440000, 0; 
0.800000, 1.440000, 0; 
1.120000, 1.440000, 0; 
1.440000, 1.440000, 0;'

redMap: '-1.440000, -1.440000, 0; 
-1.120000, -1.440000, 0; 
-0.800000, -1.440000, 0; 
-0.480000, -1.440000, 0; 
-0.160000, -1.440000, 0; 
0.160000, -1.440000, 0; 
0.480000, -1.440000, 0; 
0.800000, -1.440000, 0; 
1.120000, -1.440000, 0; 
1.440000, -1.440000, 0;'

whiteMap: '-1.440000, -1.120000, 0;
-1.440000, -0.800000, 0;
-1.440000, -0.480000, 0;
-1.440000, -0.160000, 0;
-1.440000, 0.160000, 0;
-1.440000, 0.480000, 0;
-1.440000, 0.800000, 0;
-1.440000, 1.120000, 0;
-1.120000, -1.120000, 0;
-1.120000, -0.800000, 0;
-1.120000, -0.480000, 0;
-1.120000, -0.160000, 0;
-1.120000, 0.160000, 0;
-1.120000, 0.480000, 0;
-1.120000, 0.800000, 0;
-1.120000, 1.120000, 0;
-0.800000, -1.120000, 0;
-0.800000, -0.800000, 0;
-0.800000, -0.480000, 0;
-0.800000, -0.160000, 0;
-0.800000, 0.160000, 0;
-0.800000, 0.480000, 0;
-0.800000, 0.800000, 0;
-0.800000, 1.120000, 0;
-0.480000, -1.120000, 0;
-0.480000, -0.800000, 0;
-0.480000, -0.480000, 0;
-0.480000, -0.160000, 0;
-0.480000, 0.160000, 0;
-0.480000, 0.480000, 0;
-0.480000, 0.800000, 0;
-0.480000, 1.120000, 0;
-0.160000, -1.120000, 0;
-0.160000, -0.800000, 0;
-0.160000, -0.480000, 0;
-0.160000, -0.160000, 0;
-0.160000, 0.160000, 0;
-0.160000, 0.480000, 0;
-0.160000, 0.800000, 0;
-0.160000, 1.120000, 0;
0.160000, -1.120000, 0;
0.160000, -0.800000, 0;
0.160000, -0.480000, 0;
0.160000, -0.160000, 0;
0.160000, 0.160000, 0;
0.160000, 0.480000, 0;
0.160000, 0.800000, 0;
0.160000, 1.120000, 0;
0.480000, -1.120000, 0;
0.480000, -0.800000, 0;
0.480000, -0.480000, 0;
0.480000, -0.160000, 0;
0.480000, 0.160000, 0;
0.480000, 0.480000, 0;
0.480000, 0.800000, 0;
0.480000, 1.120000, 0;
0.800000, -1.120000, 0;
0.800000, -0.800000, 0;
0.800000, -0.480000, 0;
0.800000, -0.160000, 0;
0.800000, 0.160000, 0;
0.800000, 0.480000, 0;
0.800000, 0.800000, 0;
0.800000, 1.120000, 0;
1.120000, -1.120000, 0;
1.120000, -0.800000, 0;
1.120000, -0.480000, 0;
1.120000, -0.160000, 0;
1.120000, 0.160000, 0;
1.120000, 0.480000, 0;
1.120000, 0.800000, 0;
1.120000, 1.120000, 0;
1.440000, -1.120000, 0;
1.440000, -0.800000, 0;
1.440000, -0.480000, 0;
1.440000, -0.160000, 0;
1.440000, 0.160000, 0;
1.440000, 0.480000, 0;
1.440000, 0.800000, 0;
1.440000, 1.120000, 0;'
